--- a/configure	2024-09-06 14:49:51.819932779 -0600
+++ b/configure	2024-09-06 14:53:31.953143126 -0600
@@ -3538,7 +3538,7 @@
   ac_status=$?
   printf "%s\n" "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_PSRCHIVE_LIBS=`$PKG_CONFIG --libs "psrchive >= 2024-07-25 " 2>/dev/null`
+  pkg_cv_PSRCHIVE_LIBS=`$PKG_CONFIG --libs "psrchive >= 2024-07-25 " | sed 's/-lemutls_w//' 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -3607,7 +3607,7 @@
 PSRCHIVE_ACLOCAL=`pkg-config --variable=aclocal psrchive`
 
 
-PSRPLOT_LIBS=`pkg-config --variable=psrplot_libs psrchive`
+PSRPLOT_LIBS=`pkg-config --variable=psrplot_libs psrchive | sed 's/-lemutls_w//'`
 
 
 PSRCHIVE_INCLUDE=`pkg-config --cflags-only-I psrchive`
