diff --git a/test/util/util.hpp b/test/util/util.hpp
index f012f6a5..5b845c0d 100644
--- a/test/util/util.hpp
+++ b/test/util/util.hpp
@@ -166,13 +166,13 @@ namespace util {
   std::vector<T> subtract (const std::vector<T>& a, const std::vector<T>& b);
 
   template<typename T>
-  void subtract (T* a, T* b, T* res, unsigned size);
+  void subtract (const T* a, const T* b, T* res, unsigned size);
 
   template<typename T>
   std::vector<T> abs (const std::vector<T>& a);
 
   template<typename T>
-  void abs (T* a, unsigned size);
+  void abs (const T* a, T* res, unsigned size);
 
   template<typename T>
   void load_binary_data (std::string file_path, std::vector<T>& test_data);
@@ -269,7 +269,7 @@ void test::util::load_binary_data (std::string file_path, std::vector<T>& test_d
 }
 
 template<typename T>
-void test::util::write_binary_data (std::string file_path, std::vector<T> buffer)
+void test::util::write_binary_data (std::string file_path, std::vector<T>& buffer)
 {
   test::util::write_binary_data(file_path, buffer.data(), buffer.size());
 }
